samples/bookinfo/build_push_update_images.sh

```diff
@@ -22,6 +22,7 @@ display_usage() {
     echo "	version : Version of the sample app images (Required)"
     echo "	-h|--help : Prints usage information"
     echo "	--prefix: Use the value as the prefix for image names. By default, 'istio' is used"
+    echo -e "   --multiarch-images : Enables building and pushing multiarch docker images \n\t\t\trelated to bookinfo sample apps. By default, this feature \n\t\t\tis disabled."
```

Comment:
maybe we need to add   multiarch-images  in https://github.com/istio/istio/pull/37546/files#diff-4467951412e7f755e45d7fcbfeb38a635507dd82393400eb976f65c3a1b4f993R21

Reply:
not sure what you referring to here @tanjunchen